kernel/package.json

```diff
@@ -99,6 +99,7 @@
     "cids": "^0.7.3",
     "dcl-catalyst-client": "^3.0.1",
     "dcl-crypto": "^2.2.0",
+    "dcl-quests-client": "^1.2.0",
```

Comment:
We are still using the quests client because eventually we will need it for the Quest Board (global quests list)